### PR TITLE
update base class names

### PR DESCRIPTION
**Pull request by m-nash**
# Description
<i>Add your description here!</i>

# Checklist

To ensure a quick review and merge, please ensure:
- [ ] The PR has a understandable title and description explaining the _why_ and _what_.
- [ ] The PR is opened in draft if not ready for review yet.
   - If opened in draft, please allocate sufficient time (24 hours) after moving out of draft for review
- [ ] The branch is recent enough to not have merge conflicts upon creation.

# Ready to Land?
- [ ] Build is completely green
   - Submissions with test failures require tracking issue and approval of a CODEOWNER
- [ ] At least one +1 review by a CODEOWNER
- [ ] All -1 reviews are confirmed resolved by the reviewer 
   - Override/Marking reviews stale must be discussed with CODEOWNERS first

### CODE REVIEW

File: samples/Azure.Management.Storage/Generated/StorageAccountOperations.cs [hunk 1]
```diff
@@ -45,40 +45,6 @@ protected internal StorageAccountOperations(OperationsBase options, ResourceIden
...
-                var response = _restClient.GetProperties(Id.ResourceGroupName, Id.Name, null, cancellationToken);
-                return Response.FromValue(new StorageAccount(this, response.Value), response.GetRawResponse());
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
-
```
Comment (ShivangiReja) [inline samples/Azure.Management.Storage/Generated/StorageAccountOperations.cs:81]: If we are removing this method, then I think we should pass `StorageAccountExpand? expand` parameter as optional, like [rest client's method](https://github.com/Azure/autorest.csharp/blob/8efc2364091e61ced50cec9c9305a8e5aa063d9c/samples/Azure.Management.Storage/Generated/RestOperations/StorageAccountsRestOperations.cs#L309).

Example: 
```csharp
public async virtual Task<Response<StorageAccount>> GetAsync(StorageAccountExpand? expand = null, CancellationToken cancellationToken = default)
```

Comment (m-nash) [reply samples/Azure.Management.Storage/Generated/StorageAccountOperations.cs:81]: that already exists, before we had 2 overloads, now we only have the one which is the one you mentioned on line 70

Comment (ShivangiReja) [reply samples/Azure.Management.Storage/Generated/StorageAccountOperations.cs:81]: Yes, the get method overload exists, but it takes nullable parameter(StorageAccountExpand? expand) but it should take nullable and optional parameter(StorageAccountExpand? expand = null) just like rest client method.

Comment (ShivangiReja) [reply samples/Azure.Management.Storage/Generated/StorageAccountOperations.cs:81]: Yes, the get method overload exists, but it takes nullable parameter(StorageAccountExpand? expand), now that we removed this get method, user has to always pass something in expand  parameter. To fix this, it should take nullable and optional parameter(StorageAccountExpand? expand = null) just like rest client method.

**Review by ShivangiReja — APPROVED** [review]

**Pull request merged**